package.json

```diff
@@ -5,27 +5,49 @@
   "license": "MIT",
   "repository": "datproject/dat-desktop",
   "description": "Dat Desktop App",
-  "author": "Dat Team",
+  "author": "Dat Team <community@datproject.org>",
   "scripts": {
-    "start": "electron .",
+    "start": "cross-env NODE_ENV=development electron .",
     "pretest": "npm run lint",
     "test": "npm run test:unit",
-    "test:unit": "NODE_ENV=test babel-tape-runner -r ./unit-tests/_helpers/*.js unit-tests/*.js",
+    "test:unit": "cross-env NODE_ENV=test babel-tape-runner -r ./unit-tests/_helpers/*.js unit-tests/*.js",
     "lint": "prettier-standard 'app/**/*.js' 'tests/**/*.js' 'lib/**/*.js' 'unit-tests/*.js' && standard",
-    "build": "webpack",
-    "watch": "webpack --watch"
+    "deps": "cross-env DEBUG=* dependency-check . --detective precinct --entry app/index.js",
+    "clean": "git clean -dfX",
+    "burnthemall": "rm -rf ~/.electron && npm run clean && rm -f package-lock.json && npm install",
+    "update-rdt": "rm -rf dev/react-dev-tools && ced fmkadmapgofadopljbjfkapdkoienihi dev/react-dev-tools",
+    "dev": "cross-env NODE_ENV=development sh -c \"(npm run watch &); npm run start\"",
```

Comment:
won't this lead to zombie processes? I mean how do you stop watching? 

I think that there's a way to launch electron from webpack so that we would only need `npm run start`.

Reply:
It indeed creates a zombie process. Thanks for noticing and pointing out. I am looking now into https://webpack.electron.build/ ー It would change quite a few things.